src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -278,7 +278,13 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        check(find(by.xpath("//input[@name='parameterized']")));
```

Comment:
@jglick @olivergondza sure :)

(sorry for the delay, I somehow missed the mention notification)